src/main/java/thestonedturtle/mahoganyhomes/TeleportItem.java

```diff
@@ -0,0 +1,18 @@
+package thestonedturtle.mahoganyhomes;
+
+import lombok.AllArgsConstructor;
+import lombok.Setter;
+
+@AllArgsConstructor
+@Setter
+public class TeleportItem implements Comparable<TeleportItem>
+{
+	public int ItemId;
+	public int Distance;
+
+	@Override
+	public int compareTo(TeleportItem teleportItem)
```

Comment:
this data class no longer needs to implement Comparable via comments in `TeleportItems` class